client/src/AboutUs.tsx

```diff
@@ -1,7 +1,272 @@
 import * as React from 'react';
+import Typography from '@material-ui/core/Typography';
+import { makeStyles } from '@material-ui/core/styles';
+import { Box, Grid } from '@material-ui/core';
+
+import SmallDisplayCard from './SmallDisplayCard';
+import GridImages from './GridImages';
+
+import type { Theme } from '@material-ui/core/styles';
+
+const loremIpsum = "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Facilisis placerat et, at vel tristique. Ac, gravida in quam gravida. Vel pretium nunc cursus donec enim. Sapien facilisis mauris justo, augue pharetra. Dignissim euismod fermentum sit gravida ut.";
+
+const useStyles = makeStyles((theme: Theme) => ({
+    // universal styles
+    header: {
+        fontWeight: 'bold',
+        marginBottom: 15
+    },
+    headerText: {
+        width: '100%',
+        maxWidth: '1100px',
+        fontSize: '1.3rem',
+        textAlign: 'left',
+    },
+    mainPageSection: {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        paddingTop: '50px',
+        paddingBottom: '50px',
+        paddingLeft: '20px',
+        paddingRight: '20px',
+        fontSize: '1.4rem',
+    },
+    // page specific styles
+    titleBox: {
+        height: 600,
+    },
+    whyContainer: {
+        backgroundColor: '#EBEBEB',
+        '& > *': {
+            maxWidth: '1200px',
+        },
+        '& > $header': {
+            maxWidth: '1200px',
+            width: '100%',
+            textAlign: 'left',
+        },
+        '& $headerText': {
+            textAlign: 'left',
+            width: '100%',
+        },
+    },
+    centerHorizontally:  {
+        display: 'flex',
+        flexDirection: 'column',
+        alignItems: 'center',
+        justifyContent: 'center',
+        width: '100%',
+        height: '100%',
+    },
+    whyImagePlaceholder: {
+        width: '100%',
+        maxWidth: '474px',
+        height: '300px',
+        border: '1px solid black',
+        backgroundColor: '#C4C4C4',
+    },
+    biosImagePlaceholder: {
+        width: '168px',
+        height: '168px',
+        border: '1px solid black',
+        backgroundColor: '#C4C4C4',
+    },
+    missionContainer: {
+    },
```

Comment:
unless you have plans for this lets remove
same with all other empty style objects